src/main/java/Library.java

```diff
@@ -566,4 +568,26 @@ public static int findLevenshteinDistance(String word1, String word2) {
       return stream.collect(Collectors.toList());
     }
   }
+
+  /**
+   * Convert InputStream to String.
+   *
+   * @param fileName file to read
+   * @throws IOException if an I/O error occurs
+   */
+  public static String inputStreamToString(String fileName) throws IOException {
```

Comment:
Method name indicates it would convert inputstream to string. However it takes a filename as a parameter?